lib/titiler-pgstac-api/runtime/src/.env

```diff
@@ -0,0 +1 @@
+PGSTAC_SECRET_ARN="arn:aws:secretsmanager:us-west-2:916098889494:secret:pgstac/bootstrap-pgstac-instance/093a89d4-2hB9wY"
```

Comment:
Should this be committed?  Will it be available to other accounts?

Reply:
Ugh 🤯 I did not mean to commit this.

Reply:
I removed this file (using `git rm` like you suggested), however, I don't think it entirely erases it from the history, like it still shows up in this PR. 

That said, it doesn't sound like an 'unsafe secret to leak', since it's just an ARN ? I.e it doesn't sound like I need to go and redeploy so that the ARN changes. What do you think @alukach ?

Reply:
I agree, I don't typically think of ARNs as things we need to be worried about leaking (others disagree with this view).  I think a simple `git rm` is good enough.